themes/bootstrap5/templates/Recommend/SideFacets/multiFacetsSelection.phtml

```diff
@@ -1,5 +1,9 @@
+<?php
+$attributes = $this->multiFacetsSelection === 'always' ? ' style="display: none;"' : '';
+?>
 <div class="multi-filters-selection">
-  <div>
+  <p class="multi-filters-text">Select one or more filters, then click button to apply</p>
```

Comment:
This also needs to be internationalized, so this text should be added to en.ini (you could add it, for example, `select_filters_instructions = "Select one or more filters, then click button to apply."`) and then displayed here with `<?=$this->transEsc('select_filters_instructions')?>`